ironic_prometheus_exporter/app/exporter.py

```diff
@@ -0,0 +1,30 @@
+import configparser
+import logging
+import os
+
+from flask import abort, Flask
+app = Flask(__name__)
+LOG = logging.getLogger(__name__)
+
+
+@app.route('/metrics', methods=['GET'])
+def prometheus_metrics():
+    try:
+        config = configparser.ConfigParser()
+        config.read(os.environ.get('IRONIC_CONFIG'))
+        DIR = config['oslo_messaging_notifications']['location']
+    except Exception as e:
+        LOG.error(e)
```

Comment:
Logging requires a string as a first argument. I'd suggest:
```python
LOG.exception('Unexpected error')
```
(I'm not including `e` since `LOG.exception` outputs the complete traceback).

Reply:
Ack